x-pack/test/functional/apps/monitoring/elasticsearch/nodes.js

```diff
@@ -12,7 +12,7 @@ export default function ({ getService, getPageObjects }) {
   const nodesList = getService('monitoringElasticsearchNodes');
   const esClusterSummaryStatus = getService('monitoringElasticsearchSummaryStatus');
 
-  describe('Elasticsearch nodes listing', () => {
+  describe.skip('Elasticsearch nodes listing', () => { // Skip this because page takes too long to load. Fixed in 6.4.0+
```

Comment:
Can we add this GitHub issue to a comment above it?

Reply:
There isn't a Github issue, as the fix came from React refactoring: https://github.com/elastic/x-pack-kibana/pull/5217